TFNetworkRecLayer.py

```diff
@@ -2369,9 +2369,10 @@ def _opt_search_resolve(self, layer_name, acc_ta, final_net_vars):
     from TFUtil import nd_indices, assert_min_tf_version, expand_dims_unbroadcast, get_valid_scope_name_from_str
     from TFUtil import get_shape_dim
     rec_layer = self.parent_rec_layer
-    if layer_name not in self.net.layers:  # inside loop
+    layer = self.net.get_layer(layer_name)
+    if not layer:  # inside loop
```

Comment:
This looks wrong. `self.net.get_layer` never returns `None`. If the layer does not exists, it would throw a `LayerNotFound` exception.

Reply:
Did you resolve this? Can you comment?

Reply:
Yes, it was wrong. I'm now catching `LayerNotFound` exceptions and return the original tensor array in the except block.